### PR TITLE
Document runner 2.0 feats

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Adds documentation for the new style of passing options, as well as the new multithreading feature based on bot workers.

### CODE REVIEW

File: site/docs/plugins/runner.md [hunk 1]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+grammY runner will resolve all necessary constraints on the fly and block those updates as long as necessary to ensure correct message ordering.
+
+The implementation of this is very efficient.
+It needs constant memory (unless you specify infinite concurrency), and it needs (amortized) constant processing time per update.
+
+## Graceful Shutdown
+
+In order for the bot to complete its work correctly you [should signal](../advanced/reliability.md#using-grammy-runner) it to stop when the process is about to be destroyed.
+
+Note that you can wait for the runner to terminate by `await`ing the `task` in the [RunnerHandle](https://deno.land/x/grammy_runner/mod.ts?s=RunnerHandle) returned from `run`.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:119]:
```suggestion
Note that you can wait for the runner to terminate by `await`ing the `task` in the [`RunnerHandle`](https://deno.land/x/grammy_runner/mod.ts?s=RunnerHandle) returned from `run`.
```

File: site/docs/plugins/runner.md [hunk 2]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+  console.log("Bot done processing!");
+});
+```
+
+## Advanced Options
+
+grammY runner consists of three things: a source, a runner, and a sink.
+The source pulls in updates, the sink consumes updates, and the runner configures and connects the two.
+
+> An in-depth description on how the runner workds internally can be found [down here](#how-it-works-behind-the-scenes).
```
Comment (roj1512) [inline site/docs/plugins/runner.md:134]: Workd?

File: site/docs/plugins/runner.md [hunk 3]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+run(bot, {
+  source: {},
+  runner: {},
+  sink: {},
+});
+```
+
+You should check out the `RunOptions` in the [API reference](https://deno.land/x/grammy_runner/mod.ts?s=RunOptions) to see which options are available.
+
+For example, you will find there that `allowed_updates` can be enabled using the following code snippet.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:151]:
```suggestion
For example, you will there find out that `allowed_updates` can be enabled using the following code snippet.
```

File: site/docs/plugins/runner.md [hunk 4]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+
+```ts
+run(bot, { runner: { fetch: { allowed_updates: [] } } });
+```
+
+## Multithreading
+
+JavaScript is single-threaded.
+This is amazing because [concurrency is hard](../advanced/scaling.md#concurrency-is-hard) so if there is only a single thread, then this removes a lot of headache.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:160]:
```suggestion
This is amazing because [concurrency is hard](../advanced/scaling.md#concurrency-is-hard), meaning that if there is only a single thread, a lot of headache is automatically removed.
```

File: site/docs/plugins/runner.md [hunk 5]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+
+However, if your bot has extremely high load (we are talking about 1000 updates per second and up), then doing everything on a single core might not be enough anymore.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:162]:
```suggestion
However, if your bot has an extremely high load (we are talking about 1000 updates per second and up), then doing everything on a single core might not be enough anymore.
```

File: site/docs/plugins/runner.md [hunk 6]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+Basically, a single core will start struggling with the JSON processing of all the messages your bot has to handle.
+
+### Bot Workers for Update Handling
+
+There is a simple way out: bot workers!
```
Comment (roj1512) [inline site/docs/plugins/runner.md:167]:
```suggestion
There is a simple way out: Bot workers!
```

File: site/docs/plugins/runner.md (hunk 2 above)
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:134]:
```suggestion
> An in-depth description on how the runner works internally can be found [down here](#how-it-works-behind-the-scenes).
```

File: site/docs/plugins/runner.md (hunk 4 above)
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:160]: Is automatically removed? Sounds like some technical automation

File: site/docs/plugins/runner.md (hunk 2 above)
Comment (roj1512) [reply site/docs/plugins/runner.md]: Good.

File: site/docs/plugins/runner.md (hunk 4 above)
Comment (roj1512) [reply site/docs/plugins/runner.md:160]:
```suggestion
This is amazing because [concurrency is hard](../advanced/scaling.md#concurrency-is-hard), meaning that if there is only a single thread, a lot of headache is naturally removed.
```

File: site/docs/plugins/runner.md (hunk 6 above)
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:167]: Isn't the first word capitalised iff the part following the colon is a sentence?

Comment (roj1512) [reply site/docs/plugins/runner.md:167]: Right.

File: site/docs/plugins/runner.md [hunk 7]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+
+In a sense, you can therefore specify a graph of dependencies between updates.
+grammY runner will resolve all necessary constraints on the fly and block those updates as long as necessary to ensure correct message ordering.
+
+The implementation of this is very efficient.
+It needs constant memory (unless you specify infinite concurrency), and it needs (amortized) constant processing time per update.
+
+## Graceful Shutdown
+
+In order for the bot to complete its work correctly you [should signal](../advanced/reliability.md#using-grammy-runner) it to stop when the process is about to be destroyed.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:117]:
```suggestion
In order for the bot to complete its work correctly, you [should signal it](../advanced/reliability.md#using-grammy-runner) to stop when the process is about to be destroyed.
```

File: site/docs/plugins/runner.md [hunk 8]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+const handle = run(bot);
+
+handle.task.then(() => {
+  console.log("Bot done processing!");
+});
+```
+
+## Advanced Options
+
+grammY runner consists of three things: a source, a runner, and a sink.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:131]: Maybe it’ll be cooler like this since it will match the order of the explanation following it?

```suggestion
grammY runner consists of three things: a source, a sink, and a sink.
```

File: site/docs/plugins/runner.md [hunk 9]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+2. send updates to workers              __/    /     \    \__
+                                     __/      /       \      \__
+                                    /        /         \        \
+3. process updates          BotWorker   BotWorker   BotWorker   BotWorker
+```
+
+grammY runner provides you with middleware that can send updates to bot workers.
+The bot workers can then receive this update and handle it.
+This way, the central bot only has to concern itself with pulling in and distributing updates among the bot workers it orchestrates.
+The actual update handling (filtering messages, sending replies, etc) is performed by the bot workers.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:191]:
```suggestion
The actual update handling (filtering messages, sending replies, etc.) is performed by the bot workers.
```

File: site/docs/plugins/runner.md [hunk 10]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+
+Let's now see how this can be used.
+
+### Using Bot Workers
+
+> Examples of this can be found in the [grammY runner repository](https://github.com/grammyjs/runner/tree/main/examples).
+
+We will start out by creating the central bot instance that fetches updates and distributes them among workers.
+Let's create a file called `bot.ts` with the following code.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:200]:
```suggestion
Let's start by creating a file called `bot.ts` with the following content.
```

File: site/docs/plugins/runner.md [hunk 11]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+// Create the bot.
+const bot = new Bot("");
+
+// Optionally, sequentialize updates here.
+// bot.use(sequentialize(...))
+
+// Distribute the updates among bot workers.
+bot.use(distribute(__dirname + "/worker"));
+
+// Run the concurrently with multi-threading.
```
Comment (roj1512) [inline site/docs/plugins/runner.md:219]: Run the concurrently?

File: site/docs/plugins/runner.md (hunk 8 above)
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:131]: It will have to be this:

```suggestion
grammY runner consists of three things: a source, a sink, and a runner.
```

File: site/docs/plugins/runner.md (hunk 9 above)
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:191]: TIL

File: site/docs/plugins/runner.md (hunk 11 above)
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:219]:
```suggestion
// Run the bot concurrently with multi-threading.
```

**Review by roj1512 — APPROVED** [review]
LGTM.

**Review by CikiMomogi — CHANGES_REQUESTED** [review]
Follow consistency of #664

File: site/docs/plugins/runner.md [hunk 12]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+
+<CodeGroup>
+  <CodeGroupItem title="TypeScript" active>
+
+```ts
+// worker.ts
+import { BotWorker } from "@grammyjs/runner";
+
+// Create a new bot worker.
+const bot = new BotWorker(""); // <-- pass your bot token here again
```
[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:279]: I'm not so sure about this one. We want to emphasize that the token must be passed twice. Perhaps it's good to be very different here?

File: site/docs/plugins/runner.md [hunk 13]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+
+</CodeGroupItem>
+ <CodeGroupItem title="Deno">
+
+```ts
+// worker.ts
+import { BotWorker } from "https://deno.land/x/grammy_runner/mod.ts";
+
+// Create a new bot worker.
+const bot = new BotWorker(""); // <-- pass your bot token here again
```
[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:307]: ditto

File: site/docs/plugins/runner.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (roj1512) [reply site/docs/plugins/runner.md:279]: Why not just `bot.token`?

[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:279]: It's in a different file and you really, really don't want to add the dependency because then you'd have two files that import each other.

If anything, we could consider injecting it with the first update, but this would be less explicit, harder to understand, and more complex to implement.

[the remark this answers is not in the document]
Comment (roj1512) [reply site/docs/plugins/runner.md:279]: Oh then I was fucking wrong. We just don't need the `again`.

File: site/docs/id/plugins/runner.md [hunk 14]
```diff
@@ -77,10 +79,297 @@ run(bot);
...
+
+### Bot Workers untuk Menangani Update
+
+Solusinya adalah bot workers!
+grammY runner bisa membuat beberapa workers untuk memproses update secara bersamaan (paralel) di core yang berbeda---menggunakan event loop yang berbeda serta memory yang terpisah.
+
+Di Node.js, grammY runner menggunakan [Worker Threads](https://nodejs.org/api/worker_threads.html).
+Sedangkan di Deno, grammY runner menggunakan [Web Workers](https://deno.land/manual/runtime/workers).
+
+Secara konsep, grammY runner menyediakan sebuah kelas bernama `BotWorker` yang berfungsi untuk menangani update.
```
[the remark this answers is not in the document]
Comment (fadzikri) [reply site/docs/id/plugins/runner.md:171]: Yeah, i think it's should be `class` you can search it on the search bar.

This word `kelas` should be change too. [link](https://640b1e78598018009ca31a67--grammy.netlify.app/id/plugins/chat-members.html#pengenalan)

File: site/docs/id/plugins/runner.md [hunk 15]
```diff
@@ -77,10 +79,297 @@ run(bot);
...
+bot.use(distribute(__dirname + "/worker"));
+
+// Jalankan bot secara bersamaan (concurrent) menggunakan multi-threading.
+run(bot);
+```
+
+</CodeGroupItem>
+</CodeGroup>
+
+Di direktori yang sama dengan `bot.ts`, buat file kedua dengan nama `worker.ts` (seperti yang tertera di baris ke-12 pada kode di atas).
```
Comment (fadzikri) [inline site/docs/id/plugins/runner.md:266]:
```suggestion
Pada direktori yang sama dengan `bot.ts`, buat file kedua dengan nama `worker.ts` (seperti yang tertera di baris ke-12 pada kode di atas).
```

Comment (fadzikri) [reply site/docs/id/plugins/runner.md:266]: how? but, it will be strange.

File: site/docs/id/plugins/runner.md [hunk 16]
```diff
@@ -77,10 +79,297 @@ run(bot);
...
+Secara bawaan, `distribute` membuat 4 bot workers.
+Kamu bisa mengatur jumlah tersebut dengan mudah.
+
+```ts
+// Distribusikan update ke 8 bot workers.
+bot.use(distribute(workerFile, { count: 8 }));
+```
+
+Perlu diperhatikan bahwa aplikasi kamu seharusnya tidak membuat thread lebih dari jumlah core yang tersedia di CPU kamu.
+Alih-alih meningkatkan, tindakan tersebut malah akan memperburuk performa.
```
Comment (fadzikri) [inline site/docs/id/plugins/runner.md:367]:
```suggestion
Alih-alih meningkatkan, tindakan tersebut malah akan memperburuk performa bot.
```

**Review by fadzikri — CHANGES_REQUESTED** [review]
Need review and change

File: site/docs/plugins/runner.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (roj1512) [reply site/docs/plugins/runner.md:279]: We simply won’t.

[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:279]: Can we add a note below the code snippet which points this out?

[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/plugins/runner.md:279]: Nvm, if anything, I'll do this in subsequent PQs. This is too late here

[the remark this answers is not in the document]
Comment (CikiMomogi) [reply site/docs/plugins/runner.md:279]: Agree with @KnorpelSenf. I prefer to stick with "pass your bot token here again" to make it clear the token needs to be inserted again. Since it's already explained in the previous code, it's unlikely to confuse the reader.

File: site/docs/id/plugins/runner.md [hunk 17]
```diff
@@ -77,10 +79,297 @@ run(bot);
...
+Namun, jika bot kamu memiliki beban kerja yang sangat tinggi (katakanlah 1000 update lebih per detik), memproses semuanya menggunakan satu core tidaklah cukup.
+Dengan kata lain, core yang cuma satu tersebut akan mengalami kesulitan untuk memproses semua pesan JSON yang diterima oleh bot kamu.
+
+### Bot Workers untuk Menangani Update
+
+Solusinya adalah bot workers!
+grammY runner bisa membuat beberapa workers untuk memproses update secara bersamaan (paralel) di core yang berbeda---menggunakan event loop yang berbeda serta memory yang terpisah.
+
+Di Node.js, grammY runner menggunakan [Worker Threads](https://nodejs.org/api/worker_threads.html).
+Sedangkan di Deno, grammY runner menggunakan [Web Workers](https://deno.land/manual/runtime/workers).
```
[the remark this answers is not in the document]
Comment (CikiMomogi) [reply site/docs/id/plugins/runner.md:169]: Saya merasa konteksnya jadi berubah. Ini seolah-olah mengatakan bahwa Deno menggunakan Web Workers, bukan grammy runner-nya.

[the remark this answers is not in the document]
Comment (CikiMomogi) [reply site/docs/id/plugins/runner.md:169]: Oh I see now. I did not see your comment before https://github.com/grammyjs/website/pull/623#discussion_r1132482297, now it makes sense. Perhaps we can rephrase it to "Sedangkan **di** Deno, **ia** menggunakan [Web Workers](https://deno.land/manual/runtime/workers)."

File: site/docs/id/plugins/runner.md [hunk 18]
```diff
@@ -77,10 +79,297 @@ run(bot);
...
+
+Namun, jika bot kamu memiliki beban kerja yang sangat tinggi (katakanlah 1000 update lebih per detik), memproses semuanya menggunakan satu core tidaklah cukup.
+Dengan kata lain, core yang cuma satu tersebut akan mengalami kesulitan untuk memproses semua pesan JSON yang diterima oleh bot kamu.
+
+### Bot Workers untuk Menangani Update
+
+Solusinya adalah bot workers!
+grammY runner bisa membuat beberapa workers untuk memproses update secara bersamaan (paralel) di core yang berbeda---menggunakan event loop yang berbeda serta memory yang terpisah.
+
+Di Node.js, grammY runner menggunakan [Worker Threads](https://nodejs.org/api/worker_threads.html).
```
[the remark this answers is not in the document]
Comment (CikiMomogi) [reply site/docs/id/plugins/runner.md:168]: The correct conjunction is "di", not "pada". How about this:
>"grammY runner menggunakan [Worker Threads](https://nodejs.org/api/worker_threads.html) di Node.js."

Then the next sentence would be 
>"Sedangkan di Deno, ia menggunakan [Web Workers](https://deno.land/manual/runtime/workers)."

File: site/docs/id/plugins/runner.md (hunk 15 above)
Comment (CikiMomogi) [reply site/docs/id/plugins/runner.md:266]: Definitely disagree. The correct conjunction is "di", not "pada".

File: site/docs/id/plugins/runner.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (fadzikri) [reply site/docs/id/plugins/runner.md:169]: > Saya merasa konteksnya jadi berubah. Ini seolah-olah mengatakan bahwa Deno menggunakan Web Workers, bukan grammy runner-nya.

hey, you write reply in Indonesia language 😅

File: site/docs/id/plugins/runner.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (fadzikri) [reply site/docs/id/plugins/runner.md:168]: Ok 👍🏻

File: site/docs/id/plugins/runner.md (hunk 15 above)
Comment (fadzikri) [reply site/docs/id/plugins/runner.md:266]: it's ok.

File: site/docs/id/plugins/runner.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (fadzikri) [reply site/docs/id/plugins/runner.md:169]: I've updated the suggestion.

File: site/docs/id/plugins/runner.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (fadzikri) [reply site/docs/id/plugins/runner.md:168]: I've updated the suggestion.

[the remark this answers is not in the document]
Comment (CikiMomogi) [reply site/docs/id/plugins/runner.md:168]: Nice. Thank you

File: site/docs/id/plugins/runner.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (CikiMomogi) [reply site/docs/id/plugins/runner.md:169]: > I've updated the suggestion.

Thanks

> > Saya merasa konteksnya jadi berubah. Ini seolah-olah mengatakan bahwa Deno menggunakan Web Workers, bukan grammy runner-nya.
> 
> hey, you write reply in Indonesia language :sweat_smile: 

tl;dr Sorry for using Indonesian earlier. My brain's not fully awake this morning.

Pinginnya tadi konsisten pakai bahasa Indonesia. Mana masih pagi, males mikir dua kali buat ngetranslate wkwk. Makanya pakai bahasa baku biar mereka gampang ngetranslate sendiri kalau mau tahu konteks pembicaraan kita. Tapi, dipikir-pikir akhirnya urung. Ini repo lumayan terkenal, banyak orang luar yang lihat. Terlalu egois kalau pakai bahasa Indonesia seenak jidat. Ya udah balik pakai bahasa Inggris, meskipun pakai broken inggris. #curhat

**Review by CikiMomogi — APPROVED** [review]

**Review by fadzikri — APPROVED** [review]
Good Job!

File: site/docs/plugins/runner.md [hunk 19]
```diff
@@ -126,41 +417,18 @@ For instance, it allows you start and stop it, or obtain a promise that resolves
...
-The implementation of this is very efficient.
-It needs constant memory (unless you specify infinite concurrency), and it needs (amortized) constant processing time per update.
+The `run` function does a few things to help you use the above structure with ease.
 
-## Graceful shutdown
+1. It creates an update supplier from your bot.
+2. It creates a [source](#source) from the update supplier.
+3. It creates an update consumer from your bot.
+4. It creates a [sink](#sink) from the update consumer.
+5. It creates a runner from the source and the sink.
```
Comment (habemuscode) [inline site/docs/plugins/runner.md:428]: Create link like: [runner](#runner)

File: site/docs/zh/plugins/runner.md [hunk 20]
```diff
@@ -124,43 +414,20 @@ runner 是一个简单的循环，它从 source 中拉取 updates，并将它们
...
-grammY runner 也提供了一个 `sequentialize` 中间件来实现这个目的。
-你可以查看这个 [部分](../advanced/scaling.md#并发是困难的) 来学习如何使用它。
+`run` 函数做了一些事情来帮助你轻松地使用上面的架构。
 
-我们现在将看一些更高级的使用。
+1. 它从你的 bot 创建一个 update 供应者。
+2. 它从 update 供应者创建一个 [source](#source)。
+3. 它从你的 bot 创建一个 update 消费者。
+4. 它从 update 消费者者创建一个 [sink](#sink)。
+5. 它从 source 和 sink 创建一个 runner。
```
Comment (habemuscode) [inline site/docs/zh/plugins/runner.md:427]: Create link like: [runner](#runner)

File: site/docs/id/plugins/runner.md [hunk 21]
```diff
@@ -124,40 +413,18 @@ Ketika kamu membuat sebuah runner menggunakan `createRunner` ([referensi API](ht
...
-Kemungkinan besar, kamu ingin memastikan pesan yang berasal dari chat yang sama diproses secara berurutan agar urutan pesan tidak berubah ketika [session middleware](./session.md) dipasang.
+Ada beberapa hal yang dilakukan function `run` untuk membantu kamu menggunakan struktur di atas dengan mudah, diantaranya adalah:
 
-grammY runner meng-export `sequentialize` middleware yang akan menyelesaikan masalah tersebut.
-Kamu bisa membaca [materi ini](../advanced/scaling.md#concurrency-itu-sulit) untuk mempelajari cara penggunaannya.
+1. Membuat sebuah penyuplai update dari bot kamu.
+2. Membuat sebuah [source](#source) dari penyuplai update.
+3. Membuat sebuah penerima update untuk bot kamu.
+4. Membuat sebuah [sink](#sink) dari penerima update.
+5. Membuat sebuah runner dari source dan sink.
```
Comment (habemuscode) [inline site/docs/id/plugins/runner.md:424]: Create link like: [runner](#runner)

**Review by habemuscode — APPROVED** [review]

File: site/docs/es/plugins/runner.md [hunk 22]
```diff
@@ -78,10 +80,300 @@ run(bot);
...
+There is a simple way out: bot workers!
+grammY runner lets you create several workers which can process your updates in parallel on actually different cores (using different event loops and with separate memory).
+
+On Node.js, grammY runner uses [Worker Threads](https://nodejs.org/api/worker_threads.html).
+On Deno, grammY runner uses [Web Workers](https://deno.land/manual/runtime/workers).
+
+Concuptually, grammY runner provides you with a class called `BotWorker` which can handle updates.
+It is equivalent to the regular class `Bot` (in fact, it even `extends Bot`).
+The main difference between `BotWorker` and `Bot` is that `BotWorker` cannot fetch updates.
+Instead, it has to receive them from a regular `Bot` that controls its workers.
```
Comment (CikiMomogi) [inline site/docs/es/plugins/runner.md:177]: @habemuscode untranslated section

**Review by CikiMomogi — CHANGES_REQUESTED** [review]
Untranslated section in Spanish doc

File: site/docs/id/plugins/runner.md (hunk 21 above)
Comment (CikiMomogi) [inline site/docs/id/plugins/runner.md:424]:
```suggestion
5. Membuat sebuah [runner](#runner) dari source dan sink.
```

File: site/docs/zh/plugins/runner.md (hunk 20 above)
Comment (CikiMomogi) [inline site/docs/zh/plugins/runner.md:427]:
```suggestion
5. 它从 source 和 sink 创建一个 [runner](#runner)。
```

File: site/docs/plugins/runner.md (hunk 19 above)
Comment (CikiMomogi) [inline site/docs/plugins/runner.md:428]:
```suggestion
5. It creates a [runner](#runner) from the source and the sink.
```

**Review by CikiMomogi — CHANGES_REQUESTED** [review]
Add anchor to runner

**Review by habemuscode — APPROVED** [review]

File: site/docs/plugins/runner.md [hunk 23]
```diff
@@ -78,10 +80,299 @@ run(bot);
...
+
+### Bot Workers for Update Handling
+
+There is a simple way out: bot workers!
+grammY runner lets you create several workers which can process your updates in parallel on actually different cores (using different event loops and with separate memory).
+
+On Node.js, grammY runner uses [Worker Threads](https://nodejs.org/api/worker_threads.html).
+On Deno, grammY runner uses [Web Workers](https://deno.land/manual/runtime/workers).
+
+Concuptually, grammY runner provides you with a class called `BotWorker` which can handle updates.
```
Comment (KnorpelSenf) [inline site/docs/plugins/runner.md:173]:
```suggestion
Conceptually, grammY runner provides you with a class called `BotWorker` which can handle updates.
```

Comment (habemuscode) [reply site/docs/plugins/runner.md:173]: Outdate

Comment (habemuscode) [reply site/docs/plugins/runner.md:173]: [9efa9dc](https://github.com/grammyjs/website/pull/623/commits/9efa9dcf4fa0093cc1fce192d09252354e55994f)

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you seismic much!
